vmdb/app/models/ems_refresh/parsers/openstack_common/orchestration_stacks.rb

```diff
@@ -6,6 +6,7 @@ def stack_resources(stack)
           return @resources[stack.id] if @resources && !@resources.fetch_path(stack.id).blank?
           @resources = {} unless @resources
           # TODO(lsmola) catch exception when nested-depth is not supported, it's supported from Juno OpenStack
+          # TODO(lsmola) @resources[stack.id] = stack.resources(:nested_depth => 50)
```

Comment:
@blomquisg we can merge this one first. As the subject of the PR states this is only for flat.